zapthedingbat/gov-uk-one-login-sim
# make example file names human legible

# What
include human legible description in all example files

![image](https://github.com/zapthedingbat/gov-uk-one-login-sim/assets/34273121/8d6cc02e-4aae-4325-a83d-419778fe2aca)

# How to review
- spin up locally
- ensure that the changes are ok and not excessively verbose
